tests/pgautofailover_utils.py

```diff
@@ -1060,44 +1060,40 @@ def run(self, level='-vv'):
         if not self.command:
             self.command = [self.program, 'run', '--pgdata', self.datadir, level]
 
-        self.run_proc = self.vnode.run(self.command)
+        if self.run_proc:
+            self.run_proc.release()
+        self.run_proc = self.vnode.run_unmanaged(self.command)
         print("pg_autoctl run [%d]" % self.run_proc.pid)
 
-    def execute(self, name, *args):
+    def execute(self, name, *args, timeout=COMMAND_TIMEOUT, other_node=None):
```

Comment:
I now wonder if we could give the list of other nodes at `__init__` time automatically from the node method where I think we have access to the monitor which has the list of all the data nodes anyway. Adding this `other_node=` parameter everywhere manually is becoming tiring. That said, that's material for another PR.

Reply:
Sounds good but I think indeed this fits better in a separate PR. I was thinking something along the lines of adding keeping a reference to the cluster in a node and doing all the "other_node" stuff for all of the nodes in the cluster. This would make it work for multiple stanbys too.